cmd/agent/daemon/enrichment/enrichers.go

```diff
@@ -85,15 +93,16 @@ func (enricher *fileHashEnricher) Enrich(ctx context.Context, req *EnrichedConta
 			if errors.Is(err, ErrFileDoesNotExist) {
 				// If the wanted file does not exist in the PID mount namespace, it will also not exist in the mounts of the other.
 				// We can hence simply return, as we will not find the wanted file.
-				return
+				break
```

Comment:
Why break and not return though?

Reply:
I added `enrichErrorMetric.Inc()` to the end of the function so it needs break now to be increased in this case